pkgs/os-specific/linux/fuse/common.nix

```diff
@@ -80,13 +80,10 @@ in stdenv.mkDerivation rec {
 
   doCheck = false; # v2: no tests, v3: all tests get skipped in a sandbox
 
-  postFixup = "cd $out\n" + (if isFuse3 then ''
-    install -D -m444 etc/fuse.conf $common/etc/fuse.conf
-    install -D -m444 etc/udev/rules.d/99-fuse3.rules $common/etc/udev/rules.d/99-fuse.rules
-  '' else ''
```

Comment:
Why it was removed I don't see replacement

Reply:
These `install`s are replaced by `-Dsysconfdir` in `mesonFlags`.